pom.xml

```diff
@@ -95,36 +107,31 @@
             <artifactId>httpcore</artifactId>
             <version>4.4.14</version>
         </dependency>
-        <dependency>
-            <groupId>org.jenkins-ci</groupId>
-            <artifactId>trilead-ssh2</artifactId>
```

Comment:
This didn't seem to be used anywhere. Also, there's no updated version and one of its dependencies clashes with other dependencies being brought in. I could probably just exclude that one, but then I thought removing it entirely is probably cleaner